src/__tests__/wrap.e2e.test.ts

```diff
@@ -484,12 +506,16 @@ describe("wrap", () => {
             {
               schemaPath: path.resolve(__dirname, fixtureFolderName, "invalid-schema.json"),
               version: "open-attestation/3.0",
-              unwrap: false
+              unwrap: false,
+              outputPathType: "directory"
             }
           )
         ).rejects.toThrow("Invalid schema, you must provide an $id property to your schema");
         expect(fs.readdirSync(outputDirectoryTwo)).toHaveLength(0);
       });
     });
+
+    rimraf.sync(inputDirectoryTwo);
```

Comment:
whats' this ?

Reply:
in the event that one of the tests are interrupted, i thought that this will help remove the tmp folders cleanly. but i am wrong, this is not needed

Reply:
i tested again. this does help to clear up the folder in the case that some tests fail and afterEach does not run properly

Reply:
You have problem with folder not being removed?

I dont think it's consistent enough to help on anything.

Reply:
ah i found out why. please ignore. its because i did a typescript check and generated multiple test.js files